site/layouts/shortcodes/table.html

```diff
@@ -0,0 +1,10 @@
+<!--
+# Original youtube shortcode
```

Comment:
I think this should say table shortcode rather than youtube :)

Reply:
fixed!